examples/contrib/cifar10/main.py

```diff
@@ -187,6 +187,12 @@ def run(
         **spawn_kwargs: Other kwargs to spawn run in child processes: master_addr, master_port, node_rank, nnodes
 
     """
+    # check to see if the num_epochs is greater than num_warmup_epochs
+    if num_warmup_epochs > num_epochs:
```

Comment:
@theory-in-progress @KickItLikeShika What happen if num_warmup_epochs == num_epochs ?

Reply:
Yeah, I think that's the right configuration, the original error according to @KickItLikeShika was raised when he set the ``num_epochs`` less than ``num_warmup_epochs``

Reply:
Well, even if it does not raise an error, does it make much sense to execute only warm-up phase ?

Reply:
Thanks for noting, @vfdev-5! Yup no worries about that, the scheduler can handle that.

Reply:
@KickItLikeShika can you detail on how exactly it will handle that ?

Reply:
Sure, check here https://github.com/pytorch/ignite/blob/master/ignite/handlers/param_scheduler.py#L928, we (only) check if the current miles is less than the previous one or not

Reply:
@KickItLikeShika I'm asking precisely about : 

> even if it does not raise an error, does it make much sense to execute only warm-up phase ?
For example, num_epochs=5 and num_warmup_epochs=5 --> training will be done on lr starting from 0.0 up to 0.4 and that's it.
If we expect num_epochs >= num_warmup_epochs + 1 --> training will be done on lr starting from 0.0 up to 0.4 and down to 0.0

Reply:
Sorry, the page wasn't reloaded since yesterday and didn't see your comments, IMO this error should be for the error only, here we handled raising the error incase of num_epochs is less, otherwise, the error message won't make much sense. On the other hand if we would like to inform the user about the equality of num_epochs and num_warmup_epochs we should raise a warning in this case, what do you think?

Reply:
I think it is OK to raise an error for a unrealistic use-case as well and not a warning

Reply:
Cool, @theory-in-progress please address this comment, and make sure to edit the error message as well, thanks!

Reply:
Sure!

Reply:
Sure I'll try that.

Reply:
Nope, I tried creating new venv three times, each time I get the same error for pre-commit and then  ``3 files failed to reformat`` when I run the ``run_code_style.sh fmt``